docs/admin/machine.rst

```diff
@@ -381,6 +381,32 @@ You can also specify a custom category to use `custom translator <https://learn.
    `"Authenticating with a Multi-service resource" <https://learn.microsoft.com/en-us/azure/ai-services/translator/reference/v3-0-reference#authenticating-with-a-multi-service-resource>`_
    `"Authenticating with an access token" section <https://learn.microsoft.com/en-us/azure/ai-services/translator/reference/v3-0-reference#authenticating-with-an-access-token>`_
 
+.. _mt-azure-openai:
+
+Azure OpenAI
+------------
+
+.. versionadded:: 5.8
+
+:Service ID: ``openai``
+:Configuration: +------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------+
+                | ``key``          | API key             |                                                                                                                           |
+                +------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------+
+                | ``azure_endpoint`` | Azure OpenAI Instance API URL | e.g. ``https://<my-instance>.openai.azure.com``                                                               |
+                +------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------+
+                | ``deployment``   | Deployment name     | The model's unique deployment name.                                                                                       |
+                +------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------+
+                | ``persona``      | Translator persona  | Describe the persona of translator to improve the accuracy of the translation. For example: “You are a squirrel breeder.” |
+                +------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------+
+                | ``style``        | Translator style    | Describe the style of translation. For example: “Use informal language.”                                                  |
+                +------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------+
```

Comment:
This table can be generated using `./manage.py list_machinery` so that it is in sync with the form.

Reply:
I wanted to do that myself after merging, but I forgot...